ddp-workspace/projects/ddp-rarex/src/styles/_activity.scss

```diff
@@ -363,3 +363,15 @@ p.question-sub {
   border: 3px solid mat-color($app-theme, 300);
   background: #f4efea;
 }
+
+.boolean-answer-HD_DEVELOPMENT_CONCERNS_PATIENT {
+  mat-radio-group {
+    display: flex !important;
+    flex-direction: column !important;
+  }
+  mat-radio-button {
+     margin-top: 0 !important;
+     margin-bottom: 10px !important;
+   }
```

Comment:
The one item for DDP-6046 Ticket requested that options in this question have vertical alignment.
Found that I needed to tweak margins to make options look consistent with the other ones in same survey.
Before (from ticket):
![Screen Shot 2021-05-10 at 3 46 11 PM](https://user-images.githubusercontent.com/29984380/117715944-f4855b80-b1a6-11eb-996e-acf39683102e.png)
After:
![Screen Shot 2021-05-10 at 3 47 50 PM](https://user-images.githubusercontent.com/29984380/117716068-1f6faf80-b1a7-11eb-9851-6440395a1d6a.png)